doc/requirements/srsd.rst

```diff
@@ -44,7 +44,7 @@ The database shall faithfully represent the possessed information and not alter
 The state information representing as near as possible the entirety of the run when the database write was executed, shall be retrievable to restore the previous case to a particular point in time for further use by analysts.
 
 .. req:: The database shall accept all pythonic primitive data types.
-   :id: REEQ_DB_PRIMITIVES
```

Comment:
Be more specific on what we mean by all here. What if something new is invented?

Reply:
I rewrote and removed that requirement.  There's no reason we should specify Python specifically, and we already accept non-primitive (NumPy) data in our databases.